README.md

```diff
@@ -1,4 +1,14 @@
 
+<a name="readme-top"></a>
+
+<div align="center">
+
+  <br/>
+
+  <h3><b>Math Magician</b></h3>
+
+</div>
```

Comment:
- [ ] When I click the link to the [issues page](https://github.com/Othniel2471/math-magician/blob/issues) link that you added, I get this 404 `error`. I would recommend adding the issue link in the contributing sections: 

```
## 🤝 Contributing

Contributions, issues, and feature requests are welcome!

Feel free to check the [issues page](../../issues/).
```